packages/graphback-cli/src/templates/configTemplates.ts

```diff
@@ -0,0 +1,49 @@
+import { writeFileSync } from 'fs';
+import { prompt as ask } from 'inquirer'
+import { Config } from '../templates/templateMetadata'
+
+/**
+ * Different database choices
+ */
+export const dbChoices: Config[] = [
+  {
+    name: 'postgres',
+    config: (projectName: string) => {
+      return {
+        'user': 'postgresql',
+        'password': 'postgres',
+        'database': `${projectName}`,
```

Comment:
IMHO let's hardcode name to graphback for the moment and print it. This stuff will need to change in the near future anyway so there is no point of investing time into this. 

On different operating systems, docker containers will be running under a different IP so localhost may be good for most of the cases but not always. We really need to document full flow in documentation first to make sure that we are on the same page here, before we merge that. 

PR do not need to resolve problems but I would want to avoid pushing things without having end to end vision and how we get there.